gis/gdal-grass/Portfile

```diff
@@ -1,65 +1,86 @@
 # -*- coding: utf-8; mode: tcl; tab-width: 2; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=2:ts=2:sts=2
 
 PortSystem          1.0
+PortGroup           github 1.0
 
 name                gdal-grass
 categories          gis
-version             2.2.0
-revision            2
+revision            0
+epoch               1
+
+github.setup        OSGeo gdal-grass 1.0.2
```

Comment:
This is the reason I set the `epoch`, I really hope that will be enough, as I was at least partially responsible for this decision (by not objecting).

Previously this port was part of GDAL and followed its versioning, but was separated because of circular dependency issues with GRASS. It is now a separate "addon" driver to GDAL. The idea with resetting the numbering, was to start fresh...

Reply:
> This is the reason I set the `epoch`, I really hope that will be enough

It will be enough as long as we also remember never to publish 1.4.3_0, 2.1.0_0, 2.1.0_1, 2.2.0_0, 2.2.0_1, or 2.2.0_2 again in this new version numbering system.

Reply:
> > This is the reason I set the `epoch`, I really hope that will be enough
> 
> It will be enough as long as we also remember never to publish 1.4.3_0, 2.1.0_0, 2.1.0_1, 2.2.0_0, 2.2.0_1, or 2.2.0_2 again in this new version numbering system.

Added this info in a reminder comment in Portfile. Thanks!